effect/default_effector_stream.go

```diff
@@ -0,0 +1,82 @@
+package effect
+
+// DefaultEffectorStream is the default implementation of interface EffectorStream.
+type DefaultEffectorStream struct {
+	done bool
+	res bool
+	expr string
+	idx int
+	cap int
+	expl int
```

Comment:
expl should be a slice because we can have multiple hitted policy for effect: `some(where (p_eft == allow)) && !some(where (p_eft == deny))` etc